modules/test/playwright/tests/users-admin-web/main/organizations.spec.ts

```diff
@@ -352,3 +351,96 @@ test(
 		}
 	}
 );
+
+test(
+	'Text XSS vulnerability in country and region key value',
+	{tag: '@LPD-72270'},
+	async ({
+		countriesManagementPage,
+		editCountryPage,
+		editOrganizationPage,
+		page,
+		usersAndOrganizationsPage,
+	}) => {
+		const xssString = `AnyName<img src=x onerror="alert('xss')">`;
+
+		await countriesManagementPage.goto();
+
+		const country = {
+			key: `${xssString}`,
+			number: String(getRandomInt()),
+			priority: '0',
+			threeLetterIsocode: getRandomString().substring(0, 3),
+			title: '',
+			twoLetterIsocode: getRandomString().substring(0, 2),
+		};
+
+		await expect(async () => {
+			await expect(
+				countriesManagementPage.countriesTable.searchInput
+			).toBeEditable();
+
+			await countriesManagementPage.countriesTable.newButton.click();
+
+			await expect(editCountryPage.titleInput).toBeVisible();
+		}).toPass();
+
+		await editCountryPage.editCountry(country);
+
+		await usersAndOrganizationsPage.goToOrganizations();
+
+		await usersAndOrganizationsPage.addOrganizationButton.click();
+
+		const orgName = getRandomString();
```

Comment:
create also the org with an XSS string so we can verify that also the org name is displayed correctly